### PR TITLE
move of the helloworld sample

### PR DESCRIPTION
**Pull request by mikebz**
splitting #1421 so we can go fast and have quick reviews.  This is the smallest one `helloworld` example.

The changes from alpha1 to alpha2 here is just in the Kptfile.

# testing
Testing is done by just doing `kpt pkg tree` and making sure that kpt is able to parse the package.  If folks want me to do other test cases for these packages - please let me know.

### CODE REVIEW

**Review by phanimarupaka — CHANGES_REQUESTED** [review]
README.md is important here. We need to show what is the purpose of this example and how to use it.

**Comment by mikebz** [comment]
> README.md is important here. We need to show what is the purpose of this example and how to use it.

readme updated. PTAL

File: package-examples/helloworld/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    fetching package /package-examples/helloworld from https://github.com/GoogleContainerTools/kpt to helloworld
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+  $ kpt pkg tree helloworld/
+
+    helloworld
```
Comment (phanimarupaka) [inline package-examples/helloworld/README.md:28]: This should be PKG: helloworld. Pull the latest upstream/next and try.

File: package-examples/helloworld/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  $ kpt live apply helloworld 
+
+    service/helloworld-gke created
+    deployment.apps/helloworld-gke created
+    2 resource(s) applied. 2 created, 0 unchanged, 0 configured, 0 failed
+    0 resource(s) pruned, 0 skipped, 0 failed
+
+[tree]: ../../../site/reference/pkg/tree
+[apply]: ../../../site/reference/live/init
+[apply]: ../../../site/reference/live/apply
```
Comment (phanimarupaka) [inline package-examples/helloworld/README.md:53]: Same marker is pointing to different links. Will this work ?

File: package-examples/helloworld/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# helloworld
+
+Sample helloworld package.  Showcases what a bare bones package is which doesn't do anything beyond declaring the current directory as a `kpt` package.
```
Comment (phanimarupaka) [inline package-examples/helloworld/README.md:3]: Nit: Better to have a new line after 80 chars approx. We need not add it as a lint check in future.

File: package-examples/helloworld/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld
```
Comment (phanimarupaka) [inline package-examples/helloworld/README.md:18]: Nit: Adding one more tab makes the command also fall into verbatim.

e.g. https://github.com/GoogleContainerTools/kpt/blob/80bc310b53159977701342eb939a146e02b28964/package-examples/wordpress/README.md#view-the-package-contents

File: package-examples/helloworld/README.md (hunk 2 above)
Comment (phanimarupaka) [reply package-examples/helloworld/README.md:53]: I don't see these are being referenced in the file. Were these added by mistake ? If so, please remove.

Comment (mikebz) [reply package-examples/helloworld/README.md:53]: updated, but this is modeled after your wordpress readme.
 `live init` and `live apply` are referenced in *Apply the package*

File: package-examples/helloworld/README.md (hunk 1 above)
Comment (mikebz) [reply package-examples/helloworld/README.md:28]: updated

File: package-examples/helloworld/README.md (hunk 2 above)
Comment (mikebz) [reply package-examples/helloworld/README.md:53]: I thought you wanted to add handy links for commands used in the file.

File: package-examples/helloworld/README.md (hunk 4 above)
Comment (mikebz) [reply package-examples/helloworld/README.md:18]: not sure I follow.... happy to do a GVC to figure out what's missing.

**Comment by mikebz** [comment]
Updated, one question is remaining where the feedback is not clear.  Thanks for taking a look.  This will pave the way for outlining expectations for samples going forward.

File: package-examples/helloworld/README.md (hunk 2 above)
Comment (phanimarupaka) [reply package-examples/helloworld/README.md]: These are not visible unless you reference them in the text using []. Removed them. Thanks for pointing out as there are such links in my PR too. Will clean them up.

File: package-examples/helloworld/README.md (hunk 4 above)
Comment (phanimarupaka) [reply package-examples/helloworld/README.md]: It's a minor formatting thing. Pushed a commit. You can take a look.

**Review by phanimarupaka — APPROVED** [review]

**Pull request merged**